backend/dataall/modules/s3_datasets_shares/services/share_managers/s3_bucket_share_manager.py

```diff
@@ -598,6 +599,11 @@ def handle_share_failure(self, error: Exception) -> bool:
         S3ShareAlarmService().trigger_s3_bucket_sharing_failure_alarm(
             self.target_bucket, self.share, self.target_environment
         )
+        AdminNotificationService().notify_admins_with_error_log(
+            process_error=f'Error occurred while processing s3 bucket share request for share with uri: {self.share.shareUri}',
+            process_name='s3 bucket share processor',
```

Comment:
nit: I'd probably use the classname `self.__class__.__name__` to make errors easier to trace in the codebase